lib/facter/mco_config.rb

```diff
@@ -2,7 +2,13 @@
   Facter.add("mco_#{node}_config") do
     setcode do
       config = nil
-      ["/etc/puppetlabs/mcollective/#{node}.cfg", "/etc/mcollective/#{node}.cfg"].each do |cfg|
+      if Facter.fact(:kernel).value =~ /windows/i
+        appData = Facter.fact(:common_appdata)
```

Comment:
I think @joshcooper mentioned this but it appears to have gotten lost - does this exist as a built in fact?